### ISSUE TITLE
Does avg_batch work?

### ISSUE THREAD

**Issue by AminJun** [yjlolo/vae-audio#4]
https://github.com/yjlolo/vae-audio/blob/3a43e3122da55daf89fffaf5bbf11fdb805a597f/model/loss.py#L31

It seems to me taking
`torch.sum(tensor)`
outputs a number, so taking:
`if avg_batch:
    output = torch.mean(output, dim=0)
`
becomes obsolete. Am I missing something?
